geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSDiffIntegrationTest.java

```diff
@@ -125,6 +180,19 @@ public void testSDiffStore() {
     assertThat(copyResultSet.toArray()).containsExactlyInAnyOrder((Object[]) secondSet);
   }
 
+  // One array
```

Comment:
it is not clear what the purpose of this comment is. Do you want to remove it? If not can you add an explanation.

Reply:
That was notes to myself. Removed it!